openapi.yaml

```diff
@@ -0,0 +1,761 @@
+openapi: 3.0.0
+info:
+  version: 0.0.0
+  title: Conquery API
+  description: Conquery API
+  license:
+    #TODO find a fitting license
+    name: TODO
+    url: http://TODO/
+components:
+
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+
+    Bearer:
+      type: http
+      scheme: bearer
+  parameters:
+    dataset:
+      in: path
+      name: dataset
+      required: true
+      schema:
+        $ref: "#/components/schemas/datasetId"
+    query:
+      in: path
+      name: query
+      required: true
+      schema:
+        $ref: "#/components/schemas/executionId"
+    concept:
+      in: path
+      name: concept
+      required: true
+      schema:
+        $ref: "#/components/schemas/conceptId"
+  responses:
+    unauthorized:
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+      description: The user is not logged in.
+  schemas:
+    id:
+      description: Typed identifier of an object. Usually consists of a recursive structure separated by dots, starting with the dataset. The dataset can be omitted, when the url is already in a context with a bound dataset.
+      type: string
+    tableId:
+      $ref: "#/components/schemas/id"
+    datasetId:
+      $ref: "#/components/schemas/id"
+    filterId:
+      $ref: "#/components/schemas/id"
+    selectId:
+      $ref: "#/components/schemas/id"
+    conceptId:
+      $ref: "#/components/schemas/id"
+    connectorId:
+      $ref: "#/components/schemas/id"
+    secondaryIdId:
+      $ref: "#/components/schemas/id"
+    columnId:
+      $ref: "#/components/schemas/id"
+    executionId:
+      $ref: "#/components/schemas/id"
+    userId:
+      $ref: "#/components/schemas/id"
+    dateTime: #TODO
+      type: string
+      # format: date-time TODO not compatible: Value does not conform to the required ISO-8601 datetime format. Invalid value '2022-12-13T13:33:32.455901838Z' for type datetime at
+    date: #TODO
+      type: string
+      format: date
+    query-state:
+      type: string
+      enum:
+        - NEW
+        - RUNNING
+        - FAILED
+        - DONE
+    frontend-dataset-ids:
+      type: array
+      items:
+        $ref: "#/components/schemas/frontend-dataset-id"
```

Comment:
Ich würde die Componente streichen wenn möglich und die Array notation in die Response packen. So weit ich das gesehen habe wir es nur einmal genutzt.